hazelcast/src/main/java/com/hazelcast/internal/config/DomConfigHelper.java

```diff
@@ -98,6 +98,14 @@ public static String cleanNodeName(final Node node) {
         }
         return StringUtil.lowerCaseInternal(nodeName);
     }
+    
+    public static String cleanNodeName(final Node node, final boolean shouldLowercase) {
```

Comment:
You can call this method from `cleanNodeName(Node)` with `true`, so the duplication can be removed.